rebasehelper/utils.py

```diff
@@ -141,6 +146,103 @@ def cprint(cls, message, fg=None, bg=None, style=None):
         else:
             print(message)
 
+    @staticmethod
+    def parse_rgb_device_specification(specification):
+        """Parses RGB device specification.
+
+        Args:
+            specification(str): RGB device specification.
+
+        Returns:
+            tuple: If the specification follows correct format, the first element is RGB tuple and the second is
+            bit width of the RGB. Otherwise, both elements are None.
+
+        """
+        match = re.match(r'^rgb:([A-Fa-f0-9]{1,4})/([A-Fa-f0-9]{1,4})/([A-Fa-f0-9]{1,4})$', str(specification))
+        if match:
+            rgb = match.groups()
+            bit_width = max([len(str(x)) for x in rgb]) * 4
+            return tuple([int(x, 16) for x in rgb]), bit_width
```

Comment:
I don't think it's necessary to convert the list to a tuple, but if you want to do that, you can write just `tuple(int(x, 16) for x in rgb)`.